src/coreclr/tools/Common/JitInterface/JitConfigProvider.cs

```diff
@@ -130,7 +130,8 @@ public string GetStringConfigValue(string name)
 
         private static string GetTargetSpec(TargetDetails target)
         {
-            string targetOSComponent = (target.OperatingSystem == TargetOS.Windows ? "win" : (target.OperatingSystem == TargetOS.WebAssembly ? "browser" : "unix"));
+            // Use the same Jir for browser and wasi
+            string targetOSComponent = (target.OperatingSystem == TargetOS.Windows ? "win" : (target.OperatingSystem == TargetOS.Browser || target.OperatingSystem == TargetOS.Wasi ? "browser" : "unix"));
```

Comment:
I think what we want is to use the `universal` moniker for all WASM targets, same as ARM/ARM64 do, since our code generation does not depend on the target OS (and if/when it starts, it'll be dynamically checked).

Reply:
Not sure what that actually does, but have made it the same, `universal`.